.github/workflows/backend_lint.yml

```diff
@@ -0,0 +1,32 @@
+name: Backend_Pylint_Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+  push:
+    branches:
+      - main
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
```

Comment:
@ChikangaTakudzwa Let's use flake8 for now since it is simpler to get started with. We can have a conversation about adding pylint in a separate issue.

Reply:
@ChikangaTakudzwa wonderful, thanks. This looks good.